src/styles/normalize.css

```diff
@@ -0,0 +1,7 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
It's not a good practice to use this selector.  It will be better if you will make a variable for example.